examples/CSharp/Tutorial 1.5 - Transformations/Tutorial 1.5 - Transformations.csproj

```diff
@@ -8,10 +8,13 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <PackageReference Include="Silk.NET" Version="1.1.1" />
       <PackageReference Include="SixLabors.ImageSharp" Version="1.0.0-beta0007" />
```

Comment:
Looks like this still uses beta 7 where the others use rc 3

Reply:
Fixed